lib/topological_inventory/api/error_document.rb

```diff
@@ -1,19 +1,27 @@
 module TopologicalInventory
   module Api
-    class ErrorDocument < Hash
-      def initialize
-        self["errors"] = []
-      end
-
-      def add(status, message)
+    class ErrorDocument
```

Comment:
Let's leave this the way it was for now...too much refactoring in a PR adding a new feature and it's cluttering up the diff.

Reply:
Oh I see Brandon did this in his original commits...let's leave it then...I don't want to mess with his original commits.

Reply:
ok, we can always refactor  later.